lib/service/modules/tempObjectStore.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2016 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+'use strict';
+
+module.exports = (persistentTempObjectStore) => {
```

Comment:
Node 6 supports ES6 default parameters but Node 5 doesn't. The DLC-CLI doesn't work on node 6 (yet) due to the breaking change in `Buffer`. At the moment all of the backend-sdk ES6 is within the bounds of node 5's ES6 support, and I believe it should stay this way as users occasionally run DLC services locally (in their node 5 DLC-CLI environments) to flush out syntax errors. I'll add default params to the backend-sdk after the DLC-CLI is upgraded to support node 6.x.